vite.config.ts

```diff
@@ -5,58 +5,72 @@ import pluginRewriteAll from "vite-plugin-rewrite-all";
 import { defineConfig } from "vite";
 import { svelte } from "@sveltejs/vite-plugin-svelte";
 
-export default defineConfig({
-  optimizeDeps: {
-    exclude: ["@pedrouid/environment", "@pedrouid/iso-crypto"],
-  },
-  test: {
-    deps: {
-      inline: ["@ethersproject/signing-key", "@ethersproject/basex"],
+export default defineConfig(() => {
```

Comment:
I think the approach is good. But could we do it this way instead?

```diff
diff --git a/vite.config.ts b/vite.config.ts
index c8ce07d..140561e 100644
--- a/vite.config.ts
+++ b/vite.config.ts
@@ -5,6 +5,22 @@ import pluginRewriteAll from "vite-plugin-rewrite-all";
 import { defineConfig } from "vite";
 import { svelte } from "@sveltejs/vite-plugin-svelte";

+function defineConstants() {
+  const config = {
+    VITEST: process.env.VITEST !== undefined,
+    PLAYWRIGHT: process.env.PLAYWRIGHT_TEST_BASE_URL !== undefined,
+  };
+
+  // Don't overwrite HASH_ROUTING in Playwright tests, so we can control it
+  // from within the tests.
+  if (process.env.PLAYWRIGHT_TEST_BASE_URL !== undefined) {
+    return config;
+  } else {
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    return { ...config, HASH_ROUTING: Boolean(process.env.HASH_ROUTING) };
+  }
+}
+
 export default defineConfig({
   optimizeDeps: {
     exclude: ["@pedrouid/environment", "@pedrouid/iso-crypto"],
@@ -53,10 +69,5 @@ export default defineConfig({
     },
   },

-  define: {
-    // eslint-disable-next-line @typescript-eslint/naming-convention
-    HASH_ROUTING: Boolean(process.env.HASH_ROUTING),
-    VITEST: process.env.VITEST !== undefined,
-    PLAYWRIGHT: process.env.PLAYWRIGHT_TEST_BASE_URL !== undefined,
-  },
+  define: defineConstants(),
 });
```

Reply:
d4ae9bbb